src/components/__tests__/rangeInput.test.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react"
+import { render } from "@testing-library/react"
+
+import RangeInput from "../rangeInput"
+
+const renderWrapper = ({
+  label = null,
+  name = "name",
+  minValue = null,
+  maxValue = null,
+  minPlaceholder = "min",
+  maxPlaceholder = "max",
+  handleChange = null,
+} = {}) =>
+  render(
+    <RangeInput
+      label={label}
+      name={name}
+      value={{
+        min: minValue,
+        max: maxValue,
+      }}
+      placeholder={{
+        min: minPlaceholder,
+        max: maxPlaceholder,
+      }}
+      handleChange={handleChange}
+    />
+  )
+
+describe("RangeInput", () => {
+  it("renders RangeInput correctly", () => {
+    const { container } = renderWrapper()
+    expect(container).toMatchSnapshot()
+  })
```

Comment:
I'm not sure this test brings a lot of value. The resulting markup is large and the slightest change will result in this test failing. How about checking that there are two input elements with expected names instead?